homeassistant/components/homekit/type_media_players.py

```diff
@@ -142,3 +170,211 @@ def update_state(self, new_state):
                               self.entity_id, current_state)
                 self.chars[FEATURE_TOGGLE_MUTE].set_value(current_state)
             self._flag[FEATURE_TOGGLE_MUTE] = False
+
+
+@TYPES.register('TelevisionMediaPlayer')
+class TelevisionMediaPlayer(HomeAccessory):
+    """Generate a Television Media Player accessory."""
+
+    def __init__(self, *args):
+        """Initialize a Switch accessory object."""
+        super().__init__(*args, category=CATEGORY_SWITCH)
+        self._flag = {FEATURE_ON_OFF: False, FEATURE_PLAY_PAUSE: False,
+                      FEATURE_TOGGLE_MUTE: False, FEATURE_SELECT_SOURCE: False}
+        self.chars = {FEATURE_ON_OFF: None, FEATURE_PLAY_PAUSE: None,
+                      FEATURE_TOGGLE_MUTE: None, FEATURE_SELECT_SOURCE: None,
+                      FEATURE_VOLUME_STEP: None}
+        feature_list = self.config[CONF_FEATURE_LIST]
```

Comment:
If we don't require the `feature_list` as per my first comment, this block (or most of it) is probably unnecessary.